src/skin/legacy/tooltips.cpp

```diff
@@ -451,11 +451,15 @@ void Tooltips::addStandardTooltips() {
         <<tr("Toggle visibility of Rate Control");
 
     // Used in cue/hotcue/loop tooltips below.
-    QString quantizeSnap = tr("If quantize is enabled, snaps to the nearest beat.");
+    QString quantizeSnap = tr("If snap is enabled, snaps to the nearest beat.");
```

Comment:
```suggestion
    QString snapDescription = tr("If snap is enabled, snaps to the nearest beat.");
```
This variable name is confusing now.